src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp

```diff
@@ -758,6 +758,12 @@ static void test12_printMessagesDetailsTest()
 {
     mwctst::TestHelper::printTestName("PRINT MESSAGE DETAILS TEST");
 
+#if defined(BSLS_PLATFORM_OS_SOLARIS) || defined(BSLS_PLATFORM_OS_AIX)
+    s_ignoreCheckDefAlloc = true;
+    // Disable default allocator check for this test until we can debug
+    // it on AIX/Solaris
+#endif
```

Comment:
I am not sure what code do allocations wrongly, it is difficult to find without means to build and test it on AIX/Solaris. I reviewed the storage tool and improved many things related to allocations, but maybe we should just disable this check until we debug it on the needed platforms

Reply:
This makes sense to me.  It's at least better to do this test by default on Linux, which will give us at least some early indications with GH Actions.